code/processes/housekeeping.q

```diff
@@ -50,7 +50,20 @@ csvloader:{[CSV]
 		//-if correctly columned csv has nulls, report error and skip lines 
 		[if[(any nullcheck:any null (housekeepingcsv.function;housekeepingcsv.age))>0; .lg.o[`housekeeping;"Null values found in file, skipping line(s)  ", ("," sv (string where nullcheck))]];
 		housekeepingcsv2:(housekeepingcsv[where not nullcheck]);
-			wrapper each housekeepingcsv2]]}
+			wrapper each housekeepingcsv2]];
+		
+	 //-list all the compressed file and save in a table
+	hdbfilepaths: filepaths where filepaths like "*database20*";
+	files: asc last each ("/" vs' hdbfilepaths);
+    filehandlesall: asc hsym `$hdbfilepaths;
+	compressedfiles: files where 0 < count each {-21!x} each filehandlesall;
+	compressedfilehandles: filehandlesall where 0 < count each {-21!x} each filehandlesall;
+
+	CompressedTable::(flip enlist[`files]!enlist`$compressedfiles)!{-21!x} each compressedfilehandles;
```

Comment:
Can you avoid the global amend `::`?

Reply:
Thanks, I would like to store that table globally, so one can retrieve anytime

Reply:
Use `set` or put it in a namespace to make it global.

Reply:
Do you mean using set to save on disk?